examples/schemapack/invalid/RelationClassNotFoundError.schemapack.yaml

```diff
@@ -1,10 +1,15 @@
-schemapack: 0.1.0
+schemapack: 0.2.0
 classes:
```

Comment:
This is essentially the name of the file spelled out and does not have any additional information right? See my comment above. If you think the RelationClassNotFoundError is not descriptive enough let's first think of how to improve this name so that it stands for it self.